oneview_redfish_toolkit/tests/blueprints/test_chassis.py

```diff
@@ -196,7 +197,41 @@ def test_enclosure_env_config_unexpected_error(self):
     # Blade     #
     #############
 
-    def test_get_blade_chassis(self):
+    def test_get_blade_chassis_without_computer_system(self):
+        """"Tests BladeChassis with a known Server Hardware"""
+
+        server_hardware = copy.deepcopy(self.server_hardware)
+        server_hardware["serverProfileUri"] = None
+
+        self.oneview_client.index_resources.get_all.return_value = \
+            [{"category": "server-hardware"}]
+        self.oneview_client.server_hardware.get.return_value = server_hardware
```

Comment:
I think we could have assert_called_with like [here](https://github.com/HewlettPackard/oneview-redfish-toolkit/blob/aebbe4c5af57a2625592b00af8a1e147551f0e11/oneview_redfish_toolkit/tests/blueprints/test_computer_system.py#L102)